### DISCUSSION TITLE
Slickgrid-universal standalone

### DISCUSSION THREAD

**Discussion by mboulou** [ghiscoding/slickgrid-universal#765] [Q&A]
Is there any way to download version for JS only and include it in my current Jquery/Bootstrap App?

Also any examples to use it from plain JS ?

Thanks!
M.

**Comment by ghiscoding** [comment]
Well that's the lib, you can use WebPack, Vite (like this [Vite demo](https://github.com/ghiscoding/slickgrid-universal-vite-demo)), or anything to `import` the modern JS way and you're good to go. If you really want to go the old legacy way, then use [SlickGrid](https://github.com/6pac/SlickGrid) directly. It's time to embrace the modern way. 

There's probably a way to use the lib with `<script>` import rather than `import` but I don't know or remember the way to do it. Actually you might have more luck with the way we've done it in Salesforce, it's a single all-in-one JS file, which exposes the code on the window object `window.Slicker`, read the [Wiki](https://github.com/ghiscoding/slickgrid-universal/wiki/Installation---Salesforce-(LWC))
